web_src/css/form.css

```diff
@@ -309,7 +309,8 @@ textarea:focus,
   .user.reset.password form .inline.field > textarea,
   .user.link-account form .inline.field > textarea,
   .user.signin form .inline.field > textarea,
-  .user.signup form .inline.field > textarea {
+  .user.signup form .inline.field > textarea,
+  .oauth-login-link {
```

Comment:
It doesn't look good to me to mix this `oauth-login-link ` with other "form elements".

Actually, I think you only need to do:

```
.oauth-login-link {
   width: 100%;
   max-width: ....;
}
```

Reply:
I think the problem is that these "form" styles are not well designed, so maybe it's better to keep this at the moment.